docs/public/_headers

```diff
@@ -7,6 +7,9 @@
 /favicon.ico
   Content-Type: image/x-icon
 
+/experiments/*
+  X-Robots-Tag: noindex
```

Comment:
Same as what Netlify do for its PR preview to prevent Google from indexing them.